test/generalized-rf-multipoles-missalignments/fort.3

```diff
@@ -0,0 +1,74 @@
+GEOMETRYFILE Long FODO-cell for SixTrack tests
+PRINT INPUT

```

Comment:
Please use the `SETTINGS` block in new tests, so that global debug can be enabled. All new tests should have:

```
SETTINGS
    PRINT
    DEBUG
NEXT
```

at the top instead of the `PRINT` block.